### PR TITLE
Enable MG step in azure pipeline checker.

### PR DESCRIPTION
**Pull request by liushilongbuaa**
#### Why I did it
Creates a manifest.json and bsi.json for all the files in a folder.
This generated manifest can be used to validate the contents of the folder in the future.

##### Work item tracking
- Microsoft ADO **(number only)**:  29207108

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: .azure-pipelines/azure-pipelines-build.yml [hunk 1]
```diff
@@ -186,3 +186,13 @@ jobs:
           fi
         displayName: "Build sonic image"
       - template: .azure-pipelines/check-dirty-version.yml@buildimage
+      - task: ManifestGeneratorTask@0
+        condition: ne(variables.SKIP_MG, 'y')
```
Comment (xumia) [inline .azure-pipelines/azure-pipelines-build.yml:190]: We need to add a condition only for Official build, right?

Comment (liushilongbuaa) [reply .azure-pipelines/azure-pipelines-build.yml:190]: We can set SKIP_MG in PR validation. This is convenient for test.

**Review by xumia — APPROVED** [review]
LGTM

**Pull request merged**